### PR TITLE
Added Binary Search

### PR DESCRIPTION
**Pull request by kchatr**

### CODE REVIEW

**Comment by iluwatar** [comment]
First of all apologies. It seems that email notifications haven't been working for this repository and that's why I haven't received info about opened pull requests. I'll look into this shortly.

**Comment by iluwatar** [comment]
@kchatr please add a comment to let me know when you're ready for another review

File: README.md [hunk 1]
```diff
@@ -116,6 +117,28 @@ public static void bubbleSort(int[] arr) {
...
+      int cur_guess = arr[mid];
+      if(cur_guess == item) {
+        return mid;
+      } else if(cur_guess > item) {
+        high = mid - 1;
+      } else {
+        low = mid + 1;
+      }
+    }
+    return -1;
```
Comment (charlesfinley) [inline README.md:137]: Why does this return -1? It should return the index of the value for which you are searching.

Comment (kchatr) [reply README.md:137]: Because the value wasn't found in the array.

**Comment by kchatr** [comment]
I'll be creating a new pull request, as it will be much cleaner and easier to sort through any issues!

Thanks @iluwatar and @xdvrx1 for your comments. I look forward to contributing to this repository :)

**Pull request closed without merging**